src/components/GameView/GameDialogs.vue

```diff
@@ -69,23 +69,13 @@ export default {
   },
   emits: ['clear-selection', 'handle-error'],
   computed: {
-    ...mapState({
-      playingFromDeck: ({ game }) => game.playingFromDeck,
-    }),
-
-    ...mapGetters([
-      'discarding',
-      'player',
-      'opponent',
-      'opponentPointTotal',
-      'opponentQueenCount',
-      'playerWins',
-    ]),
+    ...mapStores(useGameStore),
+    ...mapStores(useAuthStore),
     game() {
       // TODO: Figure out a better way to do this, mapping the whole module is a
       // bit unusual-- the usages can probably be changed to only need a subset,
       // or moved to some sort of store method or getter
-      return this.$store.state.game;
+      return this.gameStore;
```

Comment:
It feels a little weird to have this computed prop just to alias the gameStore that we already mapped. How about instead we replace the references to `game` with `gameStore`?